simcon_project/templates/template_management/main_view.html

```diff
@@ -10,77 +10,91 @@
         <script src="{% static 'js/jquery.bootstrap.modal.forms.js' %}"></script>
```

Comment:
theres a file thats empty... i cant comment on it... its buttons.html and its changed from delete or more tempate button.html

Reply:
just make sure to delete the file

Reply:
I think it's just saying that the file's been renamed